README.md

```diff
@@ -133,54 +133,62 @@ npm run make:all
 ## ✈️ Release
 
 1. Create `release/vX.Y.Z` branch.
-2. Update `CHANGELOG.md`.  
-   1. If a built-in Talk version is to be changed - add a note:
+2. Go to https://github.com/nextcloud/talk-desktop/releases/new and draft a new release
+   1. Choose a tag: Type the new version tag `v$(version)`
+   2. Previous tag: Select the last release
+   3. Generate release notes and copy the input
+3. Update `CHANGELOG.md` with:
+   1. The output of the previous step
+   2. If a built-in Talk version is to be changed - add a note:
       ```md
       ### Build-in Talk update
 
-      Built-in Talk in binaries is updated to $(VERSION) Talk changelog: https://github.com/nextcloud/spreed/blob/master/CHANGELOG.md
-      ``` 
-3. Update `package.json`:  
-   - For minor update:
-     ```sh
-     npm version minor
-     ```
-   - For patch update:
-     ```sh
-     npm version patch
-     ```
-4. Create **a release PR**.
-5. Merge **the release PR**.
-6. Create and push **a tag**:
-   ```sh
-   git tag -a v$(version) -m "Tagging the $(version) release."
-   git push origin v$(version)
-   git push releases v$(version)
-   ```
-7. **Draft a new release** on GitHub in [nextcloud-releases/talk-desktop](https://github.com/nextcloud-releases/talk-desktop/releases)
+      Built-in Talk in binaries is updated to $(talkVersion) Talk changelog: https://github.com/nextcloud/spreed/blob/$(talkVersion)/CHANGELOG.md
+      ```
+   3. Commit `CHANGELOG.md`
+4. Update version in `package.json` and `package-locked.json` using npm:
+   1. For minor update:
+      ```sh
+      npm version minor --no-git-tag
+      ```
+   2. For patch update:
+      ```sh
+      npm version patch --no-git-tag
+      ```
+   3. Commit `package.json` and `package-locked.json`
+5. Push your branch and create a PR.
+   1. Get approvals and merge the release PR.
+6. Pull the branch you merged into
+7. Create and push **a tag**:
+    ```sh
+    git tag -a v$(version) -m "Tagging the $(version) release."
+    git push origin v$(version)
+    git push releases v$(version)
+    ```
+8. **Draft a new release** on GitHub in [nextcloud-releases/talk-desktop](https://github.com/nextcloud-releases/talk-desktop/releases)
```

Comment:
This actually needs to be done on `nextcloud/` otherwise there is an error/warning that no PRs can be found for the commits.